### ISSUE TITLE
backslashes stripped out of strings

### ISSUE THREAD

**Issue by feross** [millermedeiros/esformatter-quotes#12]
With the following config:

```
 "quotes": {
    "type": "single",
    "avoidEscape": true
  }
```

I start with this:

`'\\"ESRI Shapefile\\"'`

If I run `esformatter` it strips off one quote, which is incorrect, and I end up with:

`'\"ESRI Shapefile\"'`

Running `esformatter` a second time, I end up with:

``'"ESRI Shapefile"'``

Original issue, for context: https://github.com/maxogden/standard-format/issues/84#issuecomment-124235400

**Comment by twolfson** [comment]
Ah, damn. I guess our RegExp isn't being respectful of leading slashes. Would you be interesting in opening a PR to patch?

**Comment by feross** [comment]
Sorry, I don't have time at the moment.

**Comment by twolfson** [comment]
Trying to clear out my queue so taking a look at this now =/

**Comment by twolfson** [comment]
Alright, a fix is up in #13. It feels like a large change so I would like another set of eyes on it.

**Comment by millermedeiros** [comment]
v1.0.3 published to npm containing the fix \o/
